erpnext/assets/doctype/asset/test_asset.py

```diff
@@ -221,7 +221,7 @@ def test_scrap_asset(self):
 			asset.precision("gross_purchase_amount"),
 		)
 		pro_rata_amount, _, _ = asset.get_pro_rata_amt(
-			asset.finance_books[0], 9000, add_months(get_last_day(purchase_date), 1), date
```

Comment:
I wanted to get the last day of the month, but the previous method doesn't work for months with 31 days.